Sources/PayPal/PayPalClient.swift

```diff
@@ -0,0 +1,79 @@
+import UIKit
+
+@_implementationOnly import PayPalCheckout
+
+#if canImport(PaymentsCore)
+import PaymentsCore
+#endif
+
+/// PayPal Paysheet to handle PayPal transaction
+public class PayPalClient {
+
+    private let config: CoreConfig
+    private let returnURL: String
+
+    // swiftlint:disable identifier_name
+    private let CheckoutFlow: PayPalUIFlow.Type
+    // swiftlint:enable identifier_name
+
+    // TODO: I don't think we can override a lazy var in our tests ... or can we?
+    lazy var nativeCheckoutSDKConfig: CheckoutConfig = {
+        CheckoutConfig(
+            clientID: config.clientID,
+            returnUrl: returnURL,
+            environment: config.environment.toNativeCheckoutSDKEnvironment()
+        )
+    }()
+
+    /// Initialize a PayPalClient to process PayPal transaction
+    /// - Parameters:
+    ///   - config: The CoreConfig object
+    ///   - returnURL: The return URL provided to the PayPal Native UI experience. Used as part of the authentication process to identify your application. This value should match the one set in the `Return URLs` section of your application's dashboard on your [PayPal developer account](https://developer.paypal.com)
+    public convenience init(config: CoreConfig, returnURL: String) {
+        self.init(config: config, returnURL: returnURL, checkoutFlow: Checkout.self)
+    }
+
+    init(config: CoreConfig, returnURL: String, checkoutFlow: PayPalUIFlow.Type) {
+        self.config = config
+        self.returnURL = returnURL
+        self.CheckoutFlow = checkoutFlow
+    }
+
+    /// Present PayPal Paysheet and start a PayPal transaction
+    /// - Parameters:
+    ///   - orderID: the order ID for the transaction
+    ///   - presentingViewController: the ViewController to present PayPalPaysheet on, if not provided, the Paysheet will be presented on your top-most ViewController
+    ///   - completion: Completion block to handle buyer's approval, cancellation, and error.
+    public func start(
```

Comment:
I think we can come up with a better name for this method. Right now we have `CardClient.approveOrder` and `PayPalClient.start` which seem disjointed.

Maybe something like `CardClient.checkout` / `PayPalClient.checkout`? We should start thinking of some good verbage / names to use throughout the SDK.

Reply:
I agree with this, and i think we should add some cohesion here between the modules.  the `CardClient` and the `PayPalClient` do different things - one is basically a wrapper for an API call, while the other initiates a user experience flow. The result ends up being the same, but do you think that we should make it clear that these are different sorts of functions in their names?

Reply:
That's a good question 🤔 . I'll let it marinate. -  I think we should talk with the Android team too on this.

Reply:
My gut says, to go with the simplest thing first, so using the same phrasing/names since the end result is the same.